pep-3104.txt

```diff
@@ -124,7 +124,7 @@ adopted.  At the time, Guido's response was:
 
     This is dangerously close to introducing CSNS [classic static
     nested scopes]. *If* you were to do so, your proposed semantics
-    of scoped seem allright. I still think there is not enough need
+    of scoped seem all right. I still think there is not enough need
```

Comment:
I would have chosen "alright", but this is quoting someone's text from elsewhere, so perhaps should remain as is (light editing may be fine).

Reply:
Ah, if it's a quote, I'll revert the change.